test/cppsuite/tests/test_live_restore.cpp

```diff
@@ -327,8 +347,12 @@ run_restore(const std::string &home, const std::string &source, const int64_t th
         connection_manager::instance().reopen(conn_config, home);
 
     auto crud_session = connection_manager::instance().create_session();
+    if (recovery)
+        configure_database(crud_session);
     if (!background_thread_mode)
         do_random_crud(crud_session, collection_count, op_count, first);
+    if (die)
```

Comment:
What's the purpose of randomly dying here?

Reply:
Dying here leaves the database in a state where it needs to perform recovery to startup. You cannot exit the test normally without shutting down WiredTiger which would mean that the next round does not need to do recovery. 
So basically if you supply the `-d` config is is expected that `-r` would come next.

Reply:
Okay makes sense, thanks